resources/indexes/HierarchicalNavigableSmallWorld.ts

```diff
@@ -305,7 +305,7 @@ export class HierarchicalNavigableSmallWorld {
 		}
 		function updateNode(id: number, node?: Node) {
 			// keep a record of all our changes, maintaining any changes that are queued to be written
-			let updatedNode: Node = updatedNodes.get(id)!;
```

Comment:
Does our linter rules do this? (I did _not_ make this change)

Reply:
Dunno, but we probably don't need it as the Map has a sensible type for the value.